command/plugin.go

```diff
@@ -196,6 +197,12 @@ var PostProcessors = map[string]packer.PostProcessor{
 	"vsphere-template":     new(vspheretemplatepostprocessor.PostProcessor),
 }
 
+var Communicators = map[string]func() packer.ConfigurableCommunicator{
+	"ssh":   func() packer.ConfigurableCommunicator { return new(communicator.SSH) },
+	"winrm": func() packer.ConfigurableCommunicator { return new(communicator.WinRM) },
+	// "none": func() packer.ConfigurableCommunicator { return new(communicator.None) },
```

Comment:
The split communicator part from builders is not there yet but parsing this is working.

Reply:
I removed that code since this is not supposed to work; will put it back later :)